enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,720 @@
+
+Note: as described [in the documentation][support-documentation], supported updates are still supported even if incoming edges are blocked, and Red Hat will eventually provide supported update paths from any supported release to the latest supported release in its z-stream.
+And, since [docs#32091][openshift-docs-32091], [that documentation][support-documentation] also points out that updates initiated after the update recommendation has been removed are still supported.
+
+Incoming bugs are evaluated to determine an impact statement based on [a generic template][rhbz-1858026-impact-statement-request].
+Some bugs only impact specific platforms, or clusters with other specific features.
+For example, rhbz#1858026 [only impacted][rhbz-1858026-impact-statement] clusters with the `None` platform which were created as 4.1 clusters and subsequently updated via 4.2, 4.3, and 4.4 to reach 4.5.
+And rhbz#1957584 [only impacted][rhbz-1957584-impact-statement] clusters updating from 4.6 to 4.7 with Routes whose `spec.host` contains no dots or was otherwise an invalid domain name.
+
+In those cases there is currently tension between wanting to protect vulnerable clusters by blocking the edge vs. wanting to avoid inconveniencing clusters which we know are not vulnerable and whose administrators may have been planning on taking the recommended update.
+This enhancement aims to reduce that tension.
+
+### Goals
+
+* [Cincinnati graph-data][graph-data] maintainers will have the ability to block edges for a vulnerable subset of clusters.
+
+### Non-Goals
+
+* Exactly scoping the set of blocked clusters to those which would have been impacted by the issue.
+    For example, some issues may be races where the impacted cluster set is a random subset of the vulnerable cluster set.
+    Any targeting of the blocked edges will reduce the number of blocked clusters which would have not been impacted, and thus reduce the contention between protecting vulnerable clusters and inconveniencing invulnerable clusters, so even overly broad scoping is better than no scoping at all.
+* Specifying a particular update service implementation.
+    This enhancement floats some ideas, but the details of the chosen approach are up to each update service's maintainers.
+
+## Proposal
+
+### Enhanced graph-data schema for blocking edges
+
+[The existing blocked-edges schema][block-edges] will be extended with the following new properties:
+
+* `url` (optional, [string][json-string]), with a URI documenting the blocking reason.
+    For example, this could link to a bug's impact statement or knowledge-base article.
+* `reason` (optional, [string][json-string]), with a CamelCase reason suitable for [a `ClusterOperatorStatusCondition` `reason` property][api-reason].
+* `message` (optional, [string][json-string]), with a human-oriented message describing the blocking reason, suitable for [a `ClusterOperatorStatusCondition` `message` property][api-message].
```

Comment:
I've pushed 43c3354 -> 0add9f4, extending the *Test Plan* and *Metadata to include when blocking a conditional request* subsections to explain the `optional` choice and also to add a formal policy that we require these in our own graph-data.

Reply:
I believe this is being done to allow flexibility in the API which out specific implementation may choose not to exercise and in order to avoid a major version bump in the API?

Reply:
OK